### DISCUSSION TITLE
Create raids

### DISCUSSION THREAD

**Discussion by lemesos** [IceWhaleTech/CasaOS#201] [Ideas]
It would be good to be able to handle the disks, for example create raids and handle them.
Why not zfs or btrfs snapshots for users files?
It would be a complete home nas server solution.

**Comment by JohnGuan** [comment]
Quick Tip: 
Please revise your idea according to the **[Ideas Guidelines](https://github.com/IceWhaleTech/CasaOS/discussions/164)** so that everyone can better understand the details and participate in the discussion.

**Comment by zimu96** [comment]
> It would be good to be able to handle the disks, for example create raids and handle them. Why not zfs or btrfs snapshots for users files? It would be a complete home nas server solution.

"A RAID solution and how to manage it would also be interesting, as I am very interested. Also, consider adding ZFS or Btrfs options for user file snapshots would be greatly appreciated. System improvements at hand."
